guide/ops/locations/_azure-ARM.md

```diff
@@ -0,0 +1,92 @@
+---
+section: Azure Compute ARM
+section_type: inline
+section_position: 2.1.1
+---
+
+### Azure Compute ARM
+
+Azure Resource Manager (ARM) is a framework for deploying and managing applications across resources and managing groups of resources as single logical units on the Microsoft Azure cloud computing platform.
+
+#### Setup the Azure credentials
+
+Firstly, install and configure Azure CLI following [these steps](http://azure.microsoft.com/en-us/documentation/articles/xplat-cli/).
+
+Using the Azure CLI, run the following commands to create a service principal
+
+    # Set mode to ARM
+    azure config mode arm
+    
+    # Enter your Microsoft account credentials when prompted
+    azure login
+    
+    # Set current subscription to create a service principal
+    azure account set <Subscription-id>
+    
+    # Create an AAD application with your information.
+    azure ad app create --name <name> --password <Password> --home-page <home-page> --identifier-uris <identifier-uris>
+    
+    # For example: azure ad app create --name "myappname"  --password abcd --home-page "https://myappwebsite" --identifier-uris "https://myappwebsite"
+    
+    # Output will include a value for `Application Id`, which will be used for the live tests
+    
+    # Create a Service Principal
+    azure ad sp create <Application-id>
+    
+    # Output will include a value for `Object Id`, to be used in the next step 
+
+
+Run the following commands to assign roles to the service principal
+
+    # Assign roles for this service principal
+    azure role assignment create --objectId <Object-id> -o Contributor -c /subscriptions/<Subscription-id>/
+
+Look up the the tenant Id
+
+    azure account show -s <Subscription-id> --json
+
+    # output will be a JSON which will include the `Tenant id`
+
+Verify service principal
+
+    azure login -u <Application-id> -p <Password> --service-principal --tenant <Tenant-id>
+
+#### Using the Azure ARM Location
+
+Below is an example Azure ARM location in YAML which will launch a Ubuntu instance in south east asia:
+
+    brooklyn.catalog:
+      id: my-azure-arm-location
+      name: "My Azure ARM location"
+      itemType: location
+      item:
+        type: jclouds:azurecompute-arm
+        brooklyn.config:
+          identity: <Application-id>
+          credential: <Password>
+          endpoint: https://management.azure.com/subscriptions/<Subscription-id>
+          oauth.endpoint: https://login.microsoftonline.com/<Tenant-id>/oauth2/token
+      
+          jclouds.azurecompute.arm.publishers: OpenLogic
+          region: southeastasia
+          loginUser: brooklyn
+          templateOptions:
+            overrideAuthenticateSudo: true 
+
+Fill the values `<Application-id>`, `<Password>`, `<Subscription-id>` and `<Tenant-id>` in from the values generated when 
```

Comment:
I'm getting weird paragraph breaks when viewing this locally. See attached screen shot. It doesn't look like the examples are in the same section as the explanation underneath them.
<img width="851" alt="screen shot 2017-01-19 at 14 56 28" src="https://cloud.githubusercontent.com/assets/3742990/22111553/8fe3c2f6-de57-11e6-8ea5-bdfdd79288da.png">